boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15-common.dtsi

```diff
@@ -61,6 +61,7 @@
 		sw2 = &button2;
 		sw3 = &button3;
 		watchdog0 = &wdt30;
+		spi-flash0 = &mx25r64;
```

Comment:
The `mx25r64` label is not defined for `cpuflpr`, so this alias should be added only for `cpuapp`.